logbus/logstreamer/logstreamer_chaos_test.go

```diff
@@ -1,231 +0,0 @@
-//go:build chaos
```

Comment:
With the introduction of the orchestrator, the cases these tests were exercising have changed / no longer exist.
It's possible the changes introduced different edge-cases and I'd like to sync with @nelsam